contributors/design-proposals/apiserver-build-in-admission-plugins.md

```diff
@@ -0,0 +1,31 @@
+# Build some Admission Controllers into the Generic API server library
```

Comment:
I think this could be the start of a general document about extending your aggregated API server and this would just be one part.